protocol/daemons/pricefeed/client/types/price_feed_mutable_market_configs.go

```diff
@@ -216,9 +238,9 @@ func (pfmmc *PricefeedMutableMarketConfigsImpl) ValidateAndTransformParams(marke
 		marketNameToId[param.Pair] = param.Id
 	}
 
-	exchangeNames := make([]ExchangeId, 0, len(pfmmc.mutableExchangeToConfigs))
+	validExchangeNames := make([]ExchangeId, 0, len(pfmmc.mutableExchangeToConfigs))
 	for exchangeName := range pfmmc.mutableExchangeToConfigs {
-		exchangeNames = append(exchangeNames, exchangeName)
+		validExchangeNames = append(validExchangeNames, exchangeName)
 	}
```

Comment:
Existing exchanges are valid exchanges. The list is named `validExchangeNames` because it's used to define which exchanges names are valid.

Still, open to renaming here if human disagreement.